src/app/modules/main/communities/module.nim

```diff
@@ -145,6 +145,11 @@ proc createMemberItem(self: Module, memberId, requestId: string, status: Members
   )
 
 method getCommunityItem(self: Module, c: CommunityDto): SectionItem =
+  # TODO: unite bannedMembers, pendingMemberRequests and declinedMemberRequests
```

Comment:
Do we have tasks for these two TODOs?

Reply:
I will add mention for [this](https://github.com/status-im/status-desktop/issues/11842) issue